packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -323,7 +322,7 @@ const Loader: React.FC<ModelLoaderProps> = React.memo(
         allFrontstages = [...frontstages];
       }
 
-      if (viewState) {
+      if (isMounted.current && viewState && viewState.iModel.isOpen) {
```

Comment:
Should also check for viewState.iModel.isBlank and proceed as necessary

Reply:
I think (hope) we can revisit the dependencies here and avoid this....ideally this would never execute for a closed connection